classes/class-wc-connect-taxjar-integration.php

```diff
@@ -50,6 +50,51 @@ public function __construct(
 		$this->cache_time = HOUR_IN_SECONDS;
 	}
 
+	/**
+	 * @param mixed  $taxjar_response
+	 * @param string $to_country
+	 * @param string $to_state
+	 *
+	 * @return string
+	 */
+	private static function generate_tax_rate_name( $taxjar_response, $to_country, $to_state ) {
```

Comment:
Why do we need `$to_country, $to_state` if `$taxjar_response` also contains these values?

Reply:
That's a good question - `to_state` might not be always present in the TaxJar response: https://d.pr/i/mYKd6V

I added both as a fallback scenario

Reply:
For the countries that don't have `$to_country, $to_state`, will they be removed form the `join`-`array_filter` logic below regardless?

Reply:
Yeah, the logic below is just for the US. But it's worth mentioning that if `$taxjar_response->jurisdictions` doesn't include `state` or `country` for whatever reason, we're also using `$to_state` or `$to_country` as a fallback.